genai-perf/genai_perf/parser.py

```diff
@@ -786,6 +811,67 @@ def _add_image_input_args(parser):
     )
 
 
+def _add_audio_input_args(parser):
```

Comment:
I know I sound like a broken clock. Alphabetize this subgroup please. Just slightly less mental load to find the options.

Reply:
Done!